pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -182,11 +179,12 @@ func (l *podNICImpl) PlugPhase1(vmi *v1.VirtualMachineInstance, iface *v1.Interf
 		return err
 	}
 
-	if err := bindMechanism.loadCachedInterface(); err != nil {
+	cachedIface, err := l.loadCachedInterface()
```

Comment:
Now that we don't need to call "loadCacheInterface" from bindMechanism we can move `bindMechanism, err := l.getPhase1Binding(vmi, iface, network, podInterfaceName, &pid)` at the place where it's used 

so
```golang
if !doesExist {
                bindMechanism, err := l.getPhase1Binding(vmi, iface, network, podInterfaceName, &pid)
	        if err != nil {
		      return err
	        }
		err = bindMechanism.discoverPodNetworkInterface()
		if err != nil {
			return err
		}
```